File: pulumi/examples/python/digitalocean_project_example.py
# Pulumi DigitalOcean Project Example (Python)
import pulumi
import pulumi_digitalocean as digitalocean

example_project = digitalocean.Project(
    "example-project",
    name="Example Project",
    purpose="Web Application",
    environment="Development",
    description="Project for IAM example"
)

pulumi.export("project_id", example_project.id)

File: pulumi-infra-project/pulumi/examples/typescript/digitalocean_project_example.ts
// Pulumi DigitalOcean Project Example (TypeScript)
import * as pulumi from "@pulumi/pulumi";
import * as digitalocean from "@pulumi/digitalocean";

const project = new digitalocean.Project("example-project", {
    name: "Example Project",
    purpose: "Web Application",
    environment: "Development",
    description: "Project for IAM example"
});

export const projectId = project.id;
